index.rst

```diff
@@ -1,61 +1,311 @@
-..
-  Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Use the following syntax for sections:
+.. sectnum::
 
-  Sections
-  ========
+.. note::
 
-  and
+   With integration of the MTAOS with the Main Telescope components approaching we have identified some design decisions made in the past that needs revisiting.
+   This technote explores some design considerations for improvements to the MTAOS operation.
 
-  Subsections
-  -----------
+The `MTAOS`_ is in charge of Main Telescope optics control loop, analyzing pairs of intra/extra focal images, computing and sending corrections for the mirror control systems (MTM1M3 and MTM2) and the Camera and M2 hexapods.
+Some background and relevant information about the MTAOS can be found in the `AOS Workshop`_ confluence page.
 
-  and
+In addition to being an extremely critical component for operating the Main Telescope (MT) the MTAOS is also unique in the sense that it interfaces with all other sub-systems, e.g.;
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+  - Camera, to acquire the intra/extra or corner rafts data for wavefront error processing.
+  - Data Management, to process the data with instrument signature removal (ISR) and wavefront estimation pipeline (WEP).
+  - Telescope and Site, mainly to apply optical corrections.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Here we explore updates to the `MTAOS (xml) interface`_ and overall software architecture to align it with the evolution of the system design.
+The most significant advancement that affect how the MTAOS operates are;
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+  - Better understanding of the observatory operation procedures.
+  - Consolidation of the data access interface, through butler gen3, for local data processing of corner wavefront sensors and remote access for full array mode.
+  - Consolidation of the OCS Controlled Pipeline System (`OCPS`_) for remote data processing.
+  - Conversion of the wavefront estimation pipeline (`WEP`_) into a butler gen3 pipeline task.
```

Comment:
You have the full-name of WEP in the front already.

Reply:
yes, but I don't want to use "wavefront estimation pipeline" all the time so I introduce the name here and can refer to it as WEP henceforth... :)

Reply:
It is a minor thing - I think TeWei is saying you introduced WEP earlier, so you can just use WEP here.

Reply:
oh! good catch! thank you...